draft-looker-oauth-jwt-cwt-status-list.md

```diff
@@ -92,11 +92,13 @@ The following rules apply to validating a JWT-based Status List Token. Applicati
 
 5. The JWT MAY contain an "exp" (expiration time) claim that conveys when it is considered expired by its issuer.
 
-6. The JWT MAY contain other claims.
+6. The JWT MAY contain a "ttl" (time to live) claim that conveys the maximum amount of time that the JWT may be cached by a client before a fresh copy is retrieved
```

Comment:
```suggestion
6. The JWT MAY contain a "ttl" (time to live) claim that conveys the maximum amount of time, in seconds, that the JWT may be cached by a client before a fresh copy is retrieved. The value of the claim MUST be a positive number.
```

Reply:
Why are we creating a new claim, that does basically the same thing as `exp` ?

Reply:
you actually approved the same idea half a year ago? The idea is that Issuers/Status List Providers may not want to resign the whole Status List if nothing has changed, therefore `ttl` communicates the update interval of the Issuer, so that RP knows how often to check for updates and `exp` communicates "do not use this list after this point of time"